Assets/HoloToolkit/Utilities/Scripts/Editor/InputManagerAxesWindow.cs

```diff
@@ -0,0 +1,295 @@
+                "Recommended\n\nUse this to get the right thumbstick's X axis, from -1 to 1 going left to right. This is used for teleporting.";
+
+            Names["CONTROLLER_RIGHT_STICK_VERTICAL"] = "CONTROLLER_RIGHT_STICK_VERTICAL";
+            Descriptions["CONTROLLER_RIGHT_STICK_VERTICAL"] =
+                "Recommended\n\nUse this to get the right thumbstick's Y axis, from -1 to 1 going bottom to top. This is used for teleporting.";
+
+            Names["CONTROLLER_RIGHT_STICK_CLICK"] = "CONTROLLER_RIGHT_STICK_CLICK";
+            Descriptions["CONTROLLER_RIGHT_STICK_CLICK"] =
+                "Recommended\n\nUse this to get the right thumbstick's clicked state.";
+
+            Names["CONTROLLER_LEFT_TRIGGER"] = "CONTROLLER_LEFT_TRIGGER";
+            Descriptions["CONTROLLER_LEFT_TRIGGER"] =
+                "Recommended\n\nUse this to get the pressed state of the left trigger, from 0 to 1 as it's pressed.";
+
+            Names["CONTROLLER_RIGHT_TRIGGER"] = "CONTROLLER_RIGHT_TRIGGER";
+            Descriptions["CONTROLLER_RIGHT_TRIGGER"] =
+                "Recommended\n\nUse this to get the pressed state of the right trigger, from 0 to 1 as it's pressed.";
+
+            Names["XBOX_SHARED_TRIGGER"] = "XBOX_SHARED_TRIGGER";
+            Descriptions["XBOX_SHARED_TRIGGER"] =
+                "Recommended\n\nUse this to get the pressed state of the both triggers, the average of two numbers: from 0 to -1 as the left trigger is pressed and from 0 to 1 as the right trigger is pressed.";
+
+            Names["XBOX_LEFT_BUMPER"] = "XBOX_LEFT_BUMPER";
+            Descriptions["XBOX_LEFT_BUMPER"] =
+                "Recommended\n\nUse this to get the pressed state of the left bumper, 0 or 1 if it's pressed.";
+
+            Names["XBOX_RIGHT_BUMPER"] = "XBOX_RIGHT_BUMPER";
+            Descriptions["XBOX_RIGHT_BUMPER"] =
+                "Recommended\n\nUse this to get the pressed state of the right bumper, 0 or 1 if it's pressed.";
+
+            Names["XBOX_A"] = "XBOX_A";
+            Descriptions["XBOX_A"] =
+                "Recommended\n\nUse this to get the pressed state of the A button, 0 or 1 if it's pressed.";
+
+            Names["XBOX_B"] = "XBOX_B";
+            Descriptions["XBOX_B"] =
+                "Recommended\n\nUse this to get the pressed state of the B button, 0 or 1 if it's pressed.";
+
+            Names["XBOX_X"] = "XBOX_X";
+            Descriptions["XBOX_X"] =
+                "Recommended\n\nUse this to get the pressed state of the X button, 0 or 1 if it's pressed.";
+
+            Names["XBOX_Y"] = "XBOX_Y";
+            Descriptions["XBOX_Y"] =
+                "Recommended\n\nUse this to get the pressed state of the Y button, 0 or 1 if it's pressed.";
+
+            Names["XBOX_VIEW"] = "XBOX_VIEW";
+            Descriptions["XBOX_VIEW"] =
+                "Recommended\n\nUse this to get the pressed state of the view button or the menu button on the left motion controller, 0 or 1 if it's pressed.";
```

Comment:
The motion controller does not have a `View` button.

Reply:
No, but it has a menu button. As mentioned above. Our currently named `XBOX_MENU` maps to the right motion controller's menu button. Our currently named `XBOX_VIEW` maps to the left motion controller's menu button. I'm sure we can come up with better names.